system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_ethereum_config.rs

```diff
@@ -42,11 +52,25 @@ pub type SnowbridgeExporter = EthereumBlobExporter<
 	EthereumSystem,
 >;
 
+pub type SnowbridgeExporterV2 = EthereumBlobExporterV2<
+	UniversalLocation,
+	EthereumNetwork,
+	snowbridge_pallet_outbound_queue_v2::Pallet<Runtime>,
+	EthereumSystemV2,
+	AssetHubParaId,
+>;
+
 parameter_types! {
 	// The gateway address is set by governance.
 	pub storage EthereumGatewayAddress: H160 = H160::zero();
 	pub AssetHubFromEthereum: Location = Location::new(1,[GlobalConsensus(RelayNetwork::get()),Parachain(polkadot_runtime_constants::system_parachain::ASSET_HUB_ID)]);
 	pub EthereumUniversalLocation: InteriorLocation = [GlobalConsensus(EthereumNetwork::get())].into();
+	pub AssetHubUniversalLocation: InteriorLocation = [GlobalConsensus(RelayNetwork::get()), Parachain(polkadot_runtime_constants::system_parachain::ASSET_HUB_ID)].into();
+	pub AssetHubLocation: Location = Location::new(1, [Parachain(polkadot_runtime_constants::system_parachain::ASSET_HUB_ID)]);
+	pub InboundQueueV2Location: InteriorLocation = [PalletInstance(InboundQueueV2PalletInstance::get())].into();
+	pub SnowbridgeFrontendLocation: Location = Location::new(1, [Parachain(polkadot_runtime_constants::system_parachain::ASSET_HUB_ID), PalletInstance(SystemFrontendPalletInstance::get())]);
+	pub AssetHubXCMFee: u128 = 1_000_000_000_000u128;
```

Comment:
Seems unused

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/796/commits/78cafa960dcb9926156a1d14102d8da08ccac8ca.